### DISCUSSION TITLE
PC/Server parts (RAM, disk, ethernet cable)

### DISCUSSION THREAD

**Discussion by zefir-git** [tailwindlabs/heroicons#604] [Ideas]
Applications about servers or PC parts would greatly benefit from an icon for RAM chip, disk/storage (HDD/SSD), and some icon to illustrate network or connectivity (e.g. RJ45 plug). `globe-alt` is somewhat suitable for the latter, however, it is more widely used to illustrate the cellular network.

**Comment by razzeee** [comment]
I'm specifically looking to replace `HddFill` and `Laptop`

**Comment by RichardOestigaard** [comment]
This would be a great addition

**Comment by henri9813** [comment]
Hello,

This idea is so nice !!!
